common/src/test/java/com/ericsson/bss/cassandra/ecaudit/common/chronicle/TestWriteReadVersionCurrent.java

```diff
@@ -102,6 +102,18 @@ public void tryReuseOnRead() throws Exception
         .withMessage("Tried to read from wire with used marshallable");
     }
 
+    @Test
+    public void testGetFieldsAvailableInRecord()
```

Comment:
Remove client parts if you agree to my comment above.

Reply:
remove this as well